src/pages/UserRegister.vue

```diff
@@ -21,6 +21,7 @@
         </b-col>
       </b-row>
       <h2 class="border-bottom my-3 pb-3">{{ $t('Register') }}</h2>
+      <b-alert :show="ferror !== ''" dismissible fade variant="danger">{{ ferror }}</b-alert>
```

Comment:
better with `v-if="featherError.length"`
What about translation of the error? we can adda custom message, like the one in UserPage
https://github.com/impresso/impresso-frontend/blob/develop/src/pages/User.vue#L27